docs(portal) session updates (#1328)
Kong/docs.konghq.com: app/enterprise/0.35-x/developer-portal/configuration/authentication/basic-auth.md

- via the [the Kong configuration file](#enable-basic-auth-via-the-kong-conf)

>**Warning** Enabling authentication in the Dev Portal requires use of the
> Sessions plugin. Developers will not be able to login if this is not set
> properly. More information about [Sessions in the Dev Portal](/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/sessions)

### Enable Portal Session Config

In the the Kong configuration file set the `portal_session_conf` property:

```
portal_session_conf={ "cookie_name": "portal_session", "secret": "CHANGE_THIS", "storage": "kong" }
```

> For testing purposes when using HTTP, config should include `"cookie_secure": false`:

Reviewer: ```suggestion
If using HTTP while testing, include `"cookie_secure": false` in the config:
```

Reviewer 2: Since this is a duplicate of the doc in Kong Manager, would you recommend the same change there? I like your change, @levfishbluefish , just don't want to have a resulting double standard. LMK and I can PR the change separately.

Reviewer: @h3xar0n yeah lets keep them consistent
